docs/style.css

```
@@ -0,0 +1,209 @@
+    --cds-border-interactive: #6929c4;
+    --cds-focus: #6929c4;
+    --cds-icon-interactive: #6929c4;
+    --cds-interactive: #6929c4;
+    --cds-link-primary: #6929c4;
+    --cds-button-primary: #6929c4;
+    --cds-button-tertiary: #6929c4;
+    --cds-background-brand: #6929c4;
+    --cds-border-interactive: #6929c4;
+    --cds-focus: #6929c4;
+    --cds-icon-interactive: #6929c4;
+    --cds-interactive: #6929c4;
+    --cds-link-primary: #6929c4;
+    --cds-button-primary: #6929c4;
+    --cds-button-tertiary: #6929c4;
+    --cds-background-brand: #6929c4;
+    --cds-focus-inverse: #6929c4;
+    --cds-link-inverse: #6929c4;
+    --cds-button-primary: #6929c4;
+    --cds-notification-action-tertiary-inverse: #6929c4;
+    --cds-background-brand: #6929c4;
+    --cds-focus-inverse: #6929c4;
+    --cds-link-inverse: #6929c4;
+    --cds-button-primary: #6929c4;
+    --cds-notification-action-tertiary-inverse: #6929c4;
+  
+    --cds-button-primary-hover: #491d8b;
+    --cds-button-tertiary-hover: #491d8b;
+    --cds-button-primary-hover: #491d8b;
+    --cds-button-tertiary-hover: #491d8b;
+    --cds-button-primary-hover: #491d8b;
+    --cds-notification-action-tertiary-inverse-hover: #491d8b;
+    --cds-button-primary-hover: #491d8b;
+    --cds-notification-action-tertiary-inverse-hover: #491d8b;
+  
+    --cds-link-primary-hover: #491d8b;
+    --cds-link-secondary: #491d8b;
+    --cds-support-info: #491d8b;
+    --cds-link-primary-hover: #491d8b;
+    --cds-link-secondary: #491d8b;
+    --cds-support-info: #491d8b;
+    --cds-highlight: #491d8b;
+    --cds-link-inverse-hover: #491d8b;
+    --cds-support-info-inverse: #491d8b;
+    --cds-tag-background-blue: #491d8b;
+    --cds-link-inverse-hover: #491d8b;
+    --cds-support-info-inverse: #491d8b;
+    --cds-tag-background-blue: #491d8b;
+   }
+
+*{
+    box-sizing: inherit;
+}
+
+html, body {
+    height: 100%;
+    margin: 0;
+    background-color: #f4f4f4;
+}
+
+body {
+    flex-flow: column nowrap;
+    font-family: 'IBM Plex Sans', sans-serif;
+}
+
+header{
+    padding: 24px 0px 24px 0px;
+    background-color: white;
+}
+
+.inside-header{
+    padding: 0px 32px 0px 32px;
+}
+
+.content{
+    padding: 24px 32px 24px 32px;
+    margin-bottom: 16px;
+}
+
+.cards{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 32px;
+    margin-top: 32px;
+    margin-bottom: 32px;
+}
+
+.card{
+    flex: 0 1 20%;
+    background-color: white;
+}
+
+cds-tile{
+    display: flex;
+    flex-flow: column;
+    gap: 16px;
+    background-color: white;
+    padding-left: 0;
+    padding-right: 0;
+}
+
+.tags{
+    display: flex;
+    flex-flow: row wrap;
+}
+
+cds-tag{
+    margin-left: 0;
+}
+
+.project-title{
+    font-size: 23px;
+    line-height: 1.25;
+    font-weight: normal;
+}
+
+.section{
+    font-weight: normal;
+    font-size: 16px;
+}
+
+p {
+    font-size: 14px;
+}
+
+.title{
+    margin: 16px 0;
+}
+
+.links{
+    display: flex;
+    flex-flow: column wrap;
+    gap: 16px;
```

```suggestion
    gap: 8px;
```
(Nitpick) I think a slightly smaller separation helps the links feel part of the same group.

---

Done!